native/membrane_video_compositor_common/src/plugins.rs

```diff
@@ -1,2 +1,29 @@
+use crate::elixir_transfer::CustomStructElixirPacket;
+
 pub mod layout;
 pub mod transformation;
+
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct PluginRegistryKey<'a>(pub &'a str);
+
+// Question for the reviewers: What should this trait be named? I don't like this name.
```

Comment:
Quite a brave proposal from me: `Renderer` :D

Reply:
I don't really like that, because even though all layouts and transformations are necessarily renderers, this trait has nothing to do with rendering. none of it's methods have anything to do with rendering, they do however have a lot to do with it being custom and pluggable.